webapp/graphite/finders/standard.py

```diff
@@ -26,6 +27,12 @@ def __init__(self, directories=None):
 
     def find_nodes(self, query):
         clean_pattern = query.pattern.replace('\\', '')
+
+        # translate query pattern if it is tagged
+        if ';' in query.pattern and not query.pattern.startswith('_tagged.'):
```

Comment:
This is a tiny unimportant detail: 
Comparing two strings from the beginning is probably faster than search through a string for a character, because if `query` is not starting with `_` it can return directly with `False`, so I'd reverse the order of these two checks.